FF1Lib/LegendaryShops.cs

```diff
@@ -91,7 +91,10 @@ public void PlaceShops()
 
 			ShopData.StoreData();
 
-			for (int i = 0; i < 8; i++) TileSets[i].StoreData();
+			for (int i = 0; i < 8; i++)

```

Comment:
I'd strongly prefer this to be a one-liner. In my opinion the extra lines can make already long code even longer and less readable.

I know a lot of this is personal preference, but at one point you have to place some trust into a programmer to code it in an understandable and readable way.